commit 7fb1f772060b7a8fa41cd813425d1b4fd611cee6
Author: Mark Robert Miller <markrmiller@apache.org>
Date:   Thu Jan 14 14:40:01 2016 +0000

    SOLR-8547: Raise time we will wait for recoveries to finish from 30 to 45 seconds.
    
    git-svn-id: https://svn.apache.org/repos/asf/lucene/dev/trunk@1724622 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/solr/core/src/test/org/apache/solr/cloud/DistribDocExpirationUpdateProcessorTest.java b/solr/core/src/test/org/apache/solr/cloud/DistribDocExpirationUpdateProcessorTest.java
index fb41793..6be317f 100644
--- a/solr/core/src/test/org/apache/solr/cloud/DistribDocExpirationUpdateProcessorTest.java
+++ b/solr/core/src/test/org/apache/solr/cloud/DistribDocExpirationUpdateProcessorTest.java
@@ -70,7 +70,7 @@ public class DistribDocExpirationUpdateProcessorTest extends AbstractFullDistrib
       indexDoc(sdoc("id", i));
     }
     commit();
-    waitForThingsToLevelOut(30);
+    waitForRecoveriesToFinish(false, 45);
 
     // this doc better not already exist
     waitForNoResults(0, params("q","id:999","rows","0","_trace","sanity_check"));
@@ -89,7 +89,6 @@ public class DistribDocExpirationUpdateProcessorTest extends AbstractFullDistrib
     waitForNoResults(180, params("q","id:999","rows","0","_trace","did_it_expire_yet"));
 
     // verify only one shard changed
-    waitForThingsToLevelOut(30);
     final Map<String,Long> finalIndexVersions = getIndexVersionOfAllReplicas();
     assertEquals("WTF? not same num versions?", 
                  initIndexVersions.size(),
